pkg/collector/corechecks/containers/kubelet/provider/pod/provider.go

```diff
@@ -120,6 +120,10 @@ func (p *Provider) generateContainerSpecMetrics(sender sender.Sender, pod *kubel
 	if len(tags) == 0 {
 		return
 	}
+	// Skip recording containers without kubelet information in tagger
+	if !isTagKeyPresent("kube_namespace", tags) {
```

Comment:
Can we add this as a constant to the file? Otherwise it will be an unnecessary allocation every time these 2 functions are called

Reply:
And maybe add a comment before explaining that these containers do not have the namespace tag, so we do not expect these metrics to be emitted?